Source/WebGPU/WebGPU/CommandEncoder.mm

```diff
@@ -184,7 +180,24 @@ static MTLStoreAction storeAction(WGPUStoreOp storeOp)
         // mtlAttachment.resolveDepthPlane = 0;
     }
 
-    // FIXME: Depth + stencil texture support
+    if (const auto* attachment = descriptor.depthStencilAttachment) {
+        const auto& mtlAttachment = mtlDescriptor.depthAttachment;
+        ASSERT("FIXME: Add support for depthReadOnly" && !attachment->depthReadOnly);
```

Comment:
Can't believe I've never thought to use ASSERTs like this (with a message)!

Reply:
oh I could have used `ASSERT_WITH_MESSAGE(` instead too, that might be more clear